BookingSystem/Frontend/src/hotelSearch/reducers/hotelSearchReducer.js

```diff
@@ -21,6 +22,7 @@ const fetchHotelsMap = {
 
 const hotelsInitialState = {
   hotels: [],
+  pageInfo: { page: 1, pageSize: config.pageSize, totalPages: 1 },
```

Comment:
hm, which rule do you follow to choose between one-line and multi-line initialization?

Reply:
linter do that